aws/src/test/java/org/apache/iceberg/aws/s3/TestS3FileIO.java

```diff
@@ -259,6 +260,17 @@ public void testFileIOJsonSerialization() {
     }
   }
 
+  @Test
+  public void testS3FileIOSerialization() throws IOException {
```

Comment:
Can you add `Kryo` to this test case name? Should we also have one for Java serialization?

Reply:
updated the test case name, added a ut for java serialization as spark test case.